### DISCUSSION TITLE
Customize Bundle Look and Feel (in Cart)

### DISCUSSION THREAD

**Discussion by horst7000** [Shopify/function-examples#274] [Feedback: Cart Transform API]
Using an app with a cart transform extension with some basic merge operations (see below), I find that the specified **_image_** and **_title_** (of the merge operation) are only visible in the checkout whereas in a cart only the **_parentVariant_** is displayed like a product. Is there a way (and if not, wouldn't it be useful) to access the underlying bundle information via liquid to customize the appearance (at least for the cart)?
```js
merge : {
  cartLines : [
    {
      cartLineId : line.id,
      quantity : 1,
    }
  ],
  parentVariantId : line.merchandise.id, // visible in cart
  title : "my bundle title", // not visible in cart
  image : { url : "https://via.placeholder.com/600/bc8f1d" },  // not visible in cart
}
```

**Comment by Hahlh** [comment]
@horst7000 
In your case I think the image is only working due to the way it is served.
It seems that Shopify always tries to alter the URL (appending for example 64x64) and in your case the image returned is still valid as the /bc8f1d part is only the color settings and the path is not getting destroyed.

This currently means that all our images are getting broken by Shopify. 
Does anybody currently have images working?

**Comment by Hahlh** [comment]
Apparently the behavior of the merge operations changed over night and now the operations throw these errors:

```javascript 

[
  "operations[0] (merge): [{code: invalid_image_url, message: The base image url must be any of the following: https://cdn.shopify.com, https://cdn.shopifycdn.net.}]",
  "operations[1] (merge): [{code: invalid_image_url, message: The base image url must be any of the following: https://cdn.shopify.com, https://cdn.shopifycdn.net.}]"
]

```

The question has now become how to use specific images for bundles at all. 
We will try to implement an upload of images via the File API into Shopify into our flow, but this might be far from optimal as they aren't directly connected to the bundles and could be deleted by accident.

**Reply by michaelhaessig** [reply]
i ran into the same issue, i've created a separate discussion here #406
